.zenodo.json

```diff
@@ -15,19 +15,13 @@
             "name": "Sai Pavan, Kamma",
             "affiliation": "University of Florida",
             "orcid": "0009-0004-4619-0409"
-        },
-        {
-            "name": "Bentz, Michael",
-            "affiliation": "University of Florida",
-            "orcid": "0000-0002-5790-4268"
```

Comment:
FYI, I reverted this. Somehow I thought Michael was not an author. He is, so I readded him before the release